airbyte-webapp/src/views/Connector/ServiceForm/components/Controls/DocumentationLink.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
+import styled from "styled-components";
+
+import { useDocumentationPanelContext } from "views/Connector/ConnectorDocumentationLayout/ConnectorDocumentationContext";
+
+const SideViewButton = styled.button`
+  cursor: pointer;
+  margin-top: 5px;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 17px;
+  text-decoration: underline;
+  display: inline-block;
+  background: none;
+  border: none;
+  padding: 0;
+
+  color: ${({ theme }) => theme.primaryColor};
+`;
+
+export const DocumentationLink: React.FC = () => {
+  const { documentationPanelOpen, setDocumentationPanelOpen } = useDocumentationPanelContext();
+
+  return (
+    <SideViewButton type="button" onClick={() => setDocumentationPanelOpen(!documentationPanelOpen)}>
+      <FormattedMessage id="form.setupGuide" />
+    </SideViewButton>
```

Comment:
This probably doesn't need to remain its own component... however I'm leaving it in place for two reasons:
(1) It's called by `<ConnectorServiceTypeControls />` and to match the pattern in that file, each field/item should be separated out
(2) Trying not to re-architect any forms with this PR as there's a full redesign of these forms in a separate issue coming up